__tests__/routes/auth.test.js

```diff
@@ -89,4 +92,35 @@ describe('auth', () => {
     expect(res.body.user).toBeDefined();
     expect(res.body.user.token).toBeDefined();
   });
+
+  test('Sigout - should return No auth token', async () => {
+    expect.assertions(3);
+    const res = await request(app)
+      .post(`${urlPrefix}/users/signout`)
+      .send();
+    expect(res.status).toBe(401);
+    expect(res.body).toBeDefined();
```

Comment:
I think this is not necessary since you are expecting for body.message for the next expectation

Reply:
@Nziranziza As we are always returning the body from each endpoint, I think you are right. Testing the body is not necessary